Tasks/PublishTestResults/make.json

```diff
@@ -0,0 +1,10 @@
+{
+  "externals": {
+    "files": [
+      {
+        "url": "https://testingpublish.blob.core.windows.net/publishcontainer/v1.0.7/TestResultsPublisher.exe",
+        "dest": "./TestResultsPublisher.exe"
```

Comment:
i would recommend using archive model. in that way if you add any extra files in future/dependencies, they will flow in here